src/stories/stubs/Dialog/MuiDialog.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import {
+  Box,
+  Button,
+  Dialog as MuiDialog,
+  DialogActions,
+  DialogContent,
+  DialogProps as MuiDialogProps,
+  DialogTitle,
+  IconButton,
+  Stack,
+} from '@mui/material'
+import { HelpPopover } from '../../../lib/containers/HelpPopover'
+import IconSvg from '../../../lib/containers/IconSvg'
+
+export interface DialogProps extends MuiDialogProps {
+  title: string
+  content: React.ReactNode
+  helpText?: string
+}
+
+export const Dialog = (props: DialogProps) => {
```

Comment:
Stub file so we can inspect a generic MUI dialog in storybook and manipulate the MUI props.